### PR TITLE
do not print-out iteration counts for GHEP SLEPc unit tests

### PR DESCRIPTION
**Pull request by davydden**
subj. 
Unfortunately, can't make it robust enough. Thus avoid it for now for generalised eigenvalue problems.

### CODE REVIEW

**Comment by bangerth** [comment]
Looks good. I think there is enough discriminating information left in the output file.

Are there second and third output files for this test that you could now remove? If no, please merge yourself. If yes, please also remove these files, and then merge.

**Comment by tjhei** [comment]
The other .output files need updating for sure because they still contain the iteration numbers.

File: tests/slepc/solve_01.cc [hunk 1]
```diff
@@ -62,8 +62,10 @@ check_solve( SolverType &solver,
       deallog << "Exception: " << e.get_exc_name() << std::endl;
     }
 
-  deallog << "Solver stopped after " << solver_control.last_step()
-          << " iterations" << std::endl;
+  // TODO make this robust on different platforms. Seems related to GHEP
+  // as solve_04 works ok.
+  //deallog << "Solver stopped after " << solver_control.last_step()
+  //        << " iterations" << std::endl;
```
Comment (tamiko) [inline tests/slepc/solve_01.cc]: Why don't you use the ```check_solver_within_range``` macro defined in tests.h?

**Comment by tamiko** [comment]
The only difference found between the different output variants is the number of iterations... So I suggest to simply remove them.

**Comment by davydden** [comment]
the other files could be removed, i will do that

File: tests/slepc/solve_01.cc (hunk 1 above)
Comment (davydden) [inline tests/slepc/solve_01.cc]: was not aware of that macro. 

I think the ranges are a bit unpredictable even though i start from the same random vectors and try to specify as much parameters as possible. I have a feeling there is still something left that SLEPc decides on its own and thus the discrepancy.

**Comment by bangerth** [comment]
Let's see how this works. We can always adjust.

**Pull request merged**